backend/tests/currentUser.spec.js

```diff
@@ -0,0 +1,75 @@
+require('dotenv').config({ path: './.env' });
+const jwt = require('jsonwebtoken');
+const { CrudServer } = require('../src/server');
+const request = require('supertest');
+const { assert, expect } = require('chai');
+const User = require('../src/api/users/user.model');
+
+describe('CurrentUser test suite', () => {
+  let server;
+
+  before(async () => {
+    const crudServer = new CrudServer();
+    await crudServer.start();
```

Comment:
Please let supertest handle `listen` execution by it own, since current implementation would cause problems right after somebody create another one file with tests

Reply:
I added to our server another method startForTest() without listening.